lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ConvertLocationAndScaleParametersToProbabilities.py

```diff
@@ -192,8 +201,9 @@ def test_threshold_below_cube(self):
             var_name="threshold").attributes['spp__relative_to_threshold'] = (
                 'below')
         expected = (np.ones((3, 3, 3)) * [0.25, 0.5, 0.75]).T
-        result = Plugin()._mean_and_variance_to_probabilities(
-            self.means, self.variances, self.template_cube)
+        result = Plugin()._location_and_scale_parameters_to_probabilities(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _location_and_scale_parameters_to_probabilities of a client class](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4576393)